### ISSUE TITLE
Use gender-neutral language when referring to users.

### ISSUE THREAD

**Issue by Julian-O** [openmediavault/openmediavault#486]
### Description of issue/question

When referring to the user, the pronoun "his" is used. It is considered more inclusive to use gender-neutral pronouns when referring to users.

(Pull request to is following shortly.)

### Steps to reproduce issue

* In the WebGUI, select Access Rights Management | Users | Add

* Note the text "Disallow the user to modify **his** account.

Note: It would be an improvement to reword this to "**Allow** the user to..." and reverse the default to avoid double-negatives, but that has impact on translations.

### Versions report

4.1.26-1

**Comment by votdev** [comment]
I thought ``his`` is gender neutral in english.

**Comment by ryecoaaron** [comment]
Nope.  Even using his/hers is not considered all inclusive.  It should be "the user update their ...".

**Comment by VoxelTek** [comment]
Unfortunately, this still seems to be present in the latest OpenMediaVault versions, as can be seen here on an instance of version 6.5.0-1. Could this be reopened, perhaps? @votdev 
![image](https://github.com/openmediavault/openmediavault/assets/53562267/b81c9ba8-6369-42e3-a309-fdd81cd65177)

**Comment by ryecoaaron** [comment]
I just did a grep of all of the source code and this seems to be the only place left.
